ch04.adoc

```diff
@@ -185,97 +187,294 @@ The `computed_standard_name` attribute indicates that the values in variable
-
-UDUNITS defines a **`year`** to be exactly 365.242198781 days (the interval between 2 successive passages of the sun through vernal equinox).
-__It is not a calendar year.__ UDUNITS defines a  **`month`** to be exactly **`year/12`**, which is __not a calendar month__.
-The CF standard follows UDUNITS in the definition of units, but we recommend that **`year`** and **`month`** should not be used, because of the potential for mistakes and confusion.
+A time coordinate is identifiable from its **`units`** alone.
+Optionally, a time coordinate may be indicated additionally by providing the **`standard_name`** attribute with an appropriate value, and/or the **`axis`** attribute with the value **`T`**.
 
 [[time-axis-ex]]
 [caption="Example 4.4. "]
-.Time axis
+.Example of a time axis
 ====
 
 ----
 double time(time) ;
-  time:long_name = "time" ;
-  time:units = "days since 1990-1-1 0:0:0" ;
+  time:axis = "T"; // optional
+  time:standard_name = "time" ; // optional
+  time:units = "days since 1990-1-1 0:0:0" ; // mandatory
 ----
 
 ====
 
-A reference time coordinate is identifiable from its units string alone.
 
-Optionally, the time coordinate may be indicated additionally by providing the **`standard_name`** attribute with an appropriate value, and/or the **`axis`** attribute with the value **`T`**.
+[[time-coordinate-units, Section 4.4.1, "Time Coordinate Units"]]
+==== Time Coordinate Units
 
-[[calendar, Section 4.4.1, "Calendar"]]
+The **`units`** attribute of a time coordinate variable takes a string value that follows the formatting requirements of the <<UDUNITS>> package (e.g. <<time-axis-ex>>).
+It must comprise a unit of measure that is physically equivalent to the SI base unit of time (i.e. the second), followed by the word **`since`** and a reference date/time.
+The format of the **`units`** string implies that the time coordinate equals the length of the time interval from the instant identified by the reference date/time to the instant identified by the time coordinate.
+This is exactly true in all cases except when leap seconds occur between the two intervals in the **`standard`**, **`proleptic_gregorian`**, and **`julian`** calendars.
+See <<leap-seconds>>.
+
+The acceptable units of measure for time are given by UDUNITS.
+The most commonly used of these strings (and their abbreviations) are **`day`** (**`d`**), **`hour`** (**`hr`**, **`h`**), **`minute`** (**`min`**) and **`second`** (**`sec`**, **`s`**).
+Plural forms are also acceptable.
+
+UDUNITS defines a **`year`** to be exactly 365.242198781 days (the interval between 2 successive passages of the sun through vernal equinox).
+__It is not a calendar year.__ UDUNITS defines a  **`month`** to be exactly **`year/12`**, which is __not a calendar month__.
+The CF standard follows UDUNITS in the definition of units, but we recommend that **`year`** and **`month`** should not be used, because of the potential for mistakes and confusion.
+
```

Comment:
Similarly, it may be worth being clear:

a **`day`** is exactly 24 hours (86400 sec). It is not a calendar day.

I suggest this because in, e.g. the Python `datetime` library, a day is a calendar day, rather than 24 hours. I *think* that only makes a difference during a DST transition, which CF doesn't allow anyway (I hope!) -- but it wouldn't hurt to be extra clear here.